<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>SkyWodd</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yet Another Fucking Phonon Music Player</string>
  </property>
  <widget class="QWidget" name="main_layout">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="global_layout">
      <item>
       <layout class="QHBoxLayout" name="main_top_layout" stretch="2,0,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QWidget" name="preview_widget" native="true">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="playlist_layout">
          <item>
           <layout class="QVBoxLayout" name="playlist_list_layout">
            <item>
             <widget class="QListView" name="playlist_list">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="playlist_controls_layout">
            <item>
             <widget class="QPushButton" name="playlist_add">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="toolTip">
               <string>Ajouter des éléments</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="playlist_remove">
              <property name="toolTip">
               <string>Enlever des éléments</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="playlist_save">
              <property name="toolTip">
               <string>Sauvegarder la playlist</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="playlist_load">
              <property name="toolTip">
               <string>Charger une playlist</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/load_file.png</normaloff>:/icons/load_file.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="music_global_layout">
        <item>
         <layout class="QHBoxLayout" name="music_time_layout">
          <item>
           <widget class="QLabel" name="music_time_elapse_label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Temps écoulé</string>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="music_time_slide">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Déplacement temporelle</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="music_time_remain_label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Temps restant</string>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="music_main_layout">
          <item>
           <layout class="QHBoxLayout" name="music_controls_layout">
            <item>
             <widget class="QPushButton" name="music_previous">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Morceau précédant</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/previous_file.png</normaloff>:/icons/previous_file.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">controls_buttons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="music_fast_rewind">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Retour rapide</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/fast_rewind.png</normaloff>:/icons/fast_rewind.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">controls_buttons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="music_play_pause">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Play</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/play_music.png</normaloff>:/icons/play_music.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">controls_buttons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="music_stop">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Stop</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/stop_playlist.png</normaloff>:/icons/stop_playlist.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">controls_buttons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="music_fast_forward">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Avance rapide</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/fast_forward.png</normaloff>:/icons/fast_forward.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">controls_buttons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="music_next">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Morceau suivant</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/next_file.png</normaloff>:/icons/next_file.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">controls_buttons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="music_random_play">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Lecture aléatoire</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/random_play.png</normaloff>:/icons/random_play.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">controls_buttons</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="music_loop_play">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="toolTip">
               <string>Lecture en boucle</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/icons/loop_play.png</normaloff>:/icons/loop_play.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">controls_buttons</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="musi_volume_controls">
            <item>
             <widget class="QSlider" name="music_volume_slide">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Volume</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="music_volume_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Volume</string>
              </property>
              <property name="text">
               <string>000%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="main_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="action_open_file"/>
    <addaction name="action_open_directory"/>
    <addaction name="separator"/>
    <addaction name="action_open_playlist"/>
    <addaction name="action_save_playlist"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menu_controls">
    <property name="title">
     <string>Lecture</string>
    </property>
    <widget class="QMenu" name="menu_loop_play">
     <property name="title">
      <string>Lecture en boucle</string>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/icons/loop_play.png</normaloff>:/icons/loop_play.png</iconset>
     </property>
     <addaction name="action_loop_playlist"/>
     <addaction name="action_loop_music"/>
    </widget>
    <addaction name="action_play_pause"/>
    <addaction name="action_stop"/>
    <addaction name="action_fast_forward"/>
    <addaction name="action_fast_rewind"/>
    <addaction name="action_next_music"/>
    <addaction name="action_previous_music"/>
    <addaction name="action_random_play"/>
    <addaction name="menu_loop_play"/>
   </widget>
   <widget class="QMenu" name="menu_audio">
    <property name="title">
     <string>Audio</string>
    </property>
    <widget class="QMenu" name="menu_audio_interfaces">
     <property name="title">
      <string>Interface audio</string>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/icons/hardware.png</normaloff>:/icons/hardware.png</iconset>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menu_supported_formats">
     <property name="title">
      <string>Formats supportés</string>
     </property>
     <property name="icon">
      <iconset resource="icons/icons.qrc">
       <normaloff>:/icons/format.png</normaloff>:/icons/format.png</iconset>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="menu_audio_interfaces"/>
    <addaction name="separator"/>
    <addaction name="menu_supported_formats"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>Vue</string>
    </property>
    <addaction name="action_enable_preview"/>
    <addaction name="action_enable_fft"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="action_manual"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_controls"/>
   <addaction name="menu_audio"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="main_statusbar"/>
  <action name="action_open_file">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/load_file.png</normaloff>:/icons/load_file.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir fichier</string>
   </property>
  </action>
  <action name="action_open_directory">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/load_directory.png</normaloff>:/icons/load_directory.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir dossier</string>
   </property>
  </action>
  <action name="action_open_playlist">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/load_playlist.png</normaloff>:/icons/load_playlist.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir playlist</string>
   </property>
  </action>
  <action name="action_save_playlist">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer playlist</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="action_play_pause">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/play_music.png</normaloff>:/icons/play_music.png</iconset>
   </property>
   <property name="text">
    <string>Play / Pause</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/stop_playlist.png</normaloff>:/icons/stop_playlist.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="action_fast_forward">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/fast_forward.png</normaloff>:/icons/fast_forward.png</iconset>
   </property>
   <property name="text">
    <string>Avance rapide</string>
   </property>
  </action>
  <action name="action_fast_rewind">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/fast_rewind.png</normaloff>:/icons/fast_rewind.png</iconset>
   </property>
   <property name="text">
    <string>Retour rapide</string>
   </property>
  </action>
  <action name="action_next_music">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/next_file.png</normaloff>:/icons/next_file.png</iconset>
   </property>
   <property name="text">
    <string>Morceau suivant</string>
   </property>
  </action>
  <action name="action_previous_music">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/previous_file.png</normaloff>:/icons/previous_file.png</iconset>
   </property>
   <property name="text">
    <string>Morceau précédant</string>
   </property>
  </action>
  <action name="action_random_play">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/random_play.png</normaloff>:/icons/random_play.png</iconset>
   </property>
   <property name="text">
    <string>Lecture aléatoire</string>
   </property>
  </action>
  <action name="actionQualit_son">
   <property name="text">
    <string>Qualité du son</string>
   </property>
  </action>
  <action name="actionEchantillonage">
   <property name="text">
    <string>Echantillonage</string>
   </property>
  </action>
  <action name="action_enable_preview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Activer la pochette</string>
   </property>
  </action>
  <action name="action_enable_fft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Activer la vue FFT</string>
   </property>
  </action>
  <action name="action_manual">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Manuel d'utilisation</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>A propos de</string>
   </property>
  </action>
  <action name="action_loop_playlist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/playlist.png</normaloff>:/icons/playlist.png</iconset>
   </property>
   <property name="text">
    <string>Playlist</string>
   </property>
  </action>
  <action name="action_loop_music">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/music.png</normaloff>:/icons/music.png</iconset>
   </property>
   <property name="text">
    <string>Morceau en cours</string>
   </property>
  </action>
  <action name="actionEffets_support_s">
   <property name="text">
    <string>Effets supportés</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_close</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="controls_buttons"/>
 </buttongroups>
</ui>
